tests/multi_ptr/multi_ptr_common.h

```diff
@@ -11,6 +11,29 @@
 
 #include "../common/common.h"
 #include "../common/type_coverage.h"
+#include "../common/type_list.h"
+
+namespace Catch {
+template <>
+struct StringMaker<sycl::access::address_space> {
+  using type = sycl::access::address_space;
+  static std::string convert(type value) {
+    switch (value) {
+      case type::global_space:
+        return "access::address_space::global_space";
+      case type::local_space:
+        return "access::address_space::local_space";
+      case type::private_space:
+        return "access::address_space::private_space";
+      case type::generic_space:
+        return "access::address_space::generic_space";
+      default:
+        // no stringification for deprecated ones
+        return "unknown address_space";
```

Comment:
```suggestion
        return "unknown address_space or adress_space is deprecated";
```
Possibly can help us in the future when some of the adress_space modes will fail to stringify.

Reply:
Good idea, changed to "unknown or deprecated address_space".